src/embed.js

```diff
@@ -274,6 +277,47 @@ function setupWeb3() {
     oauthStream.write({ name: 'oauth', data: { calledFromEmbed } })
   }
 
+  /**
+   * Expose the getPublicKey API to the Dapp through window.torus object
+   * @param {String} email Email address of the user
+   */
+
+  window.torus.getPublicKey = async function(email) {
+    // Select random node from the list of endpoints
```

Comment:
please convert this into a promise.
i.e, make this function return a promise so that users can await the result.
Also, it's recommended that we don't use async/await syntax if we are gonna publish this package so that users who don't use babel can also use the npm package.
We distribute two versions of this repo.
1. npm package containing src folder
2. create a distributable embed.min.js

Reply:
** async function (email) {} will return a promise. So await will work by default.  ** 
If we can't use async await, should i just rollback to the previous version of getPublicKey then ?